src/components/Course/Content/Content.js

```diff
@@ -8,4 +9,9 @@ const Content = ({ body, title }) => (
   </div>
 );
 
+Content.PropTypes = {
```

Comment:
In order for these prop types to work, I believe the case is important. This should be `.propTypes` instead of `.PropTypes`, and elsewhere in this MR too.

Reply:
Whoops, fixed.